conf/spark-defaults.conf.template

```diff
@@ -19,9 +19,16 @@
 # This is useful for setting default environmental settings.
 
 # Example:
-# spark.master                     spark://master:7077
-# spark.eventLog.enabled           true
-# spark.eventLog.dir               hdfs://namenode:8021/directory
-# spark.serializer                 org.apache.spark.serializer.KryoSerializer
-# spark.driver.memory              5g
-# spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+# spark.master                          spark://master:7077
+# spark.eventLog.enabled                true
+# spark.eventLog.dir                    hdfs://namenode:8021/directory
+# spark.serializer                      org.apache.spark.serializer.KryoSerializer
+# spark.driver.memory                   5g
+# spark.executor.extraJavaOptions       -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
```

Comment:
Hi, @krishna-pandey .
It seems that you can revert the change on line 22 ~ 27.
Could you re-adjust the spacing?